guides/shell_style.md

```diff
@@ -0,0 +1,45 @@
+---
+category: Bash Style
+---
+# Bash Style
+
+Where possible, we want to be consistent with our use of Bash so engineers switching betwen projects
```

Comment:
this line just trails o...

Reply:
Indeed, it's not really finished but i wanted to get some feedback on the document as a whole :)